recipe/build.sh

```diff
@@ -134,3 +135,7 @@ if [[ -d $PREFIX/lib_pypy ]]; then
 else
 	pypy3 -m compileall $PREFIX/lib/pypy${PY_VERSION} || true;
 fi
+# make sure all pypy3 processes are dead,
+# somehow zombie processes cause problems later
+pkill -9 pypy3
+pkill -9 pypy
```

Comment:
Trying this to prevent the `OSError: [Errno 26] Text file busy: '/home/conda/feedstock_root/build_artifacts/[...]/bin/pypy3'` error